Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+// When the .NET scripting backend is enabled and C# projects are built
+// Unity doesn't include the the required assemblies (i.e. the ones below).
+// Given that the .NET backend is deprecated by Unity at this point it's we have
+// to work around this on our end.
+using Microsoft.MixedReality.Toolkit.UI;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using System.Collections;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    public class PressableButtonTests
+    {
+        #region Utilities
+        private GameObject InstantiateSceneAndDefaultPressableButton()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene(true);
+            TestUtilities.InitializePlayspace();
+
+            RenderSettings.skybox = null;
+
+            Object pressableButtonPrefab = AssetDatabase.LoadAssetAtPath("Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Prefabs/PressableButton.prefab", typeof(Object));
+            GameObject testButton = Object.Instantiate(pressableButtonPrefab) as GameObject;
+
+            return testButton;
+        }
+
+        #endregion
+
+        #region Tests
+
+        [UnityTest]
+        public IEnumerator Test01_PressableButtonInstantiate()
```

Comment:
Per suggestion from @wiwei we want to avoid numbering the tests since it's arbitrary. Suggest removing the numbers to stay consistent with his suggestion.